pyproject.toml

```diff
@@ -298,12 +298,12 @@ extend-select = [
     "TRY002", # Prohibit use of `raise Exception`, use specific exceptions instead.
 ]
 ignore = [
-    "D100", # TODO: Missing docstring in public module
+    "D100", # Unwanted; Docstring at the top of every file.
     "D102", # TODO: Missing docstring in public method
     "D103", # TODO: Missing docstring in public function
-    "D104", # TODO: Missing docstring in public package
-    "D105", # Do not want.  See https://lists.apache.org/thread/8jbg1dd2lr2cfydtqbjxsd6pb6q2wkc3
-    "D107", # TODO: Missing docstring in __init__
+    "D104", # Unwanted; Docstring at the top of every `__init__.py` file.
+    "D105", # Unwanted; See https://lists.apache.org/thread/8jbg1dd2lr2cfydtqbjxsd6pb6q2wkc3
+    "D107", # Unwanted; Docstring in every constructor is unnecesary if the class has a docstring.
```

Comment:
```suggestion
    "D107", # Unwanted; Docstring in every constructor is unnecessary if the class has a docstring.
```

Reply:
Have you seen the failure or was it your Hawk-correctness-eye @jedcunningham  🙃 ?

Reply:
🙃 wasn't my eyes at this hour haha